marimo/_output/hypertext.py

```diff
@@ -133,7 +133,9 @@ def _mime_(self) -> tuple[KnownMimeType, str]:
     def __format__(self, spec: str) -> str:
         """Format `self` as HTML text"""
         del spec
-        return " ".join([line.strip() for line in self.text.split("\n")])
+        return " ".join(
+            [line.strip() for line in self.text.split("\n")]
+        ).strip()
```

Comment:
i would personally strip before splitting rather than stripping after joining

Reply:
why is that? in the case it starts/ends with a newline, then its surrounded by whitespace